Makefile

```diff
@@ -1,14 +1,17 @@
 
-SHELL  ?= $(shell command -v bash 2>/dev/null)
-DOCKER ?= $(shell command -v docker 2>/dev/null)
-PANDOC ?= $(shell command -v pandoc 2>/dev/null)
+EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
+OUTPUT_DIRNAME		?= output/
```

Comment:
On Wed, Jun 01, 2016 at 12:09:19PM -0700, Vincent Batts wrote:
> > @@ -1,14 +1,17 @@
> >  
> > -SHELL  ?= $(shell command -v bash 2>/dev/null)
> > -DOCKER ?= $(shell command -v docker 2>/dev/null)
> > -PANDOC ?= $(shell command -v pandoc 2>/dev/null)
> > +EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
> > +OUTPUT_DIRNAME		?= output/
> 
> so, not in this PR ;-)

This PR is landing the new variable.  Whether two overlapping
variables is confusing enough to be worth blocking a merge is up to
the maintainers ;).  Still unifying them shouldn't be *that* hard.
Maybe just:

  CONTAINER_OUTPUT_DIRNAME = $(OUTPUT_DIRNAME)

here,

  CONTAINER_OUTPUT_DIRNAME = /$(OUTPUT_DIRNAME)

inside the ifneq DOCKER block, and PANDOC_DST →
CONTAINER_OUTPUT_DIRNAME in the rules?